File: java/8.3/example1.java
import java.util.Scanner;

public class example1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the first number");
        Integer num1 = Integer.parseInt(scanner.nextLine());
        System.out.println("Enter the seccond number");
        Integer num2 = Integer.parseInt(scanner.nextLine());
        Integer total = num1 + num2;
        System.out.println(total);
    }
}

File: rust/src/e83/example1.rs
pub fn main() {
    let mut num1s = String::new();
    let mut num2s = String::new();
    println!("Enter a number");
    std::io::stdin().read_line(&mut num1s).unwrap();
    println!("Enter a seccond number");
    std::io::stdin().read_line(&mut num2s).unwrap();
    let num1 = num1s.trim().parse::<i32>().unwrap();
    let num2 = num2s.trim().parse::<i32>().unwrap();
    let total = num1 + num2;
    println!("The total is {}", total);
}